app/jobs/reindex_works_job.rb

```diff
@@ -1,12 +1,41 @@
 # frozen_string_literal: true
 
-# Triggers reindexing on all works
 class ReindexWorksJob < ApplicationJob
-  def perform
+
+  def perform(opts = {})
+    index_query_field = opts.dig(:index_query_field) || 'iiif_index_strategy_tesim'
+    index_query_value = opts.dig(:index_query_value) || IndexerHelper.iiif_index_strategy
+    index_file_sets = opts.dig(:index_file_sets) || true
+    async_fs_jobs = opts.dig(:async_fs_jobs) || false
+    row_count = opts.dig(:row_count) || 100
+    sleep_break = opts.dig(:row_count) || 10
```

Comment:
```suggestion
    sleep_break = opts.dig(:sleep_break) || 10
```

Reply:
New commits force pushed to resolve above.